pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -487,25 +497,37 @@ func assessNodesUsagesAndRelativeThresholds(
 		rawUsages, rawCapacities, ResourceUsageToResourceThreshold,
 	)
 
-	// calculate the average usage and then deviate it according to the
-	// user provided thresholds.
+	// calculate the average usage.
 	average := normalizer.Average(usage)
+	klog.V(3).InfoS(
+		"Assessed average usage",
+		thresholdsToKeysAndValues(average)...,
+	)
+
+	// decrease the provided threshold from the average to get the low
+	// span. also make sure the resulting values are between 0 and 100.
+	lowSpan = normalizer.Clamp(
```

Comment:
Based on https://dave.cheney.net/2017/04/30/if-a-map-isnt-a-reference-variable-what-is-it a map is not a reference. So this should not change the actual value of `lowSpan` passed by invoker.

Reply:
Done.